src/access_nri_intake/cli.py

```diff
@@ -54,10 +54,14 @@ def _parse_build_inputs(config_yamls, build_path):
         for kwargs in sources:
             source_args = config_args
 
-            source_args["path"] = kwargs.pop("path")
+            source_args["path"] = [
+                os.path.join(data_path, _) for _ in kwargs.pop("path")
```

Comment:
Not specific to this but in general we seem to be using pathlib.Path and os.path throughout this module (and package). I think we should probably move over to doing everything with pathlib.Path?

Reply:
I started playing with this, and it might be a bit of extra work - there's quite the mix of `Path` objects and path strings flying around. Plus, I see you've already made an issue for this, so I'll mark this comment as resolved and we can fix that up later.